lib/jormungandr/test/integration/Test/Integration/Jormungandr/Scenario/API/StakePools.hs

```diff
@@ -843,3 +848,10 @@ joinStakePoolWithFixtureWallet ctx = do
             [ expectFieldEqual delegation (Delegating (p ^. #id))
             ]
     return (w, p)
+
+sortByPerformance
+    :: Functor f
+    => (a, f [ApiStakePool])
+    -> (a, f [ApiStakePool])
+sortByPerformance =
+    second (fmap (sortOn (Down . view #apparentPerformance)))
```

Comment:
This should not be needed!

The problem is that the total rewards are 0 in the integration tests.

If `R` is not 0, the tests pass without `sortByPerformance`.

This highlights an interesting problem: what if the rewards *are* 0 in production?